src/common/search-engines.ts

```diff
@@ -219,6 +219,7 @@ export const searchEngineMatches: Readonly<Record<SearchEngineId, string[]>> = {
     '*://start.duckduckgo.com/?*',
   ],
   ecosia: ['https://www.ecosia.org/search?*'],
+  qwant: ['https://www.qwant.com/*', 'https://lite.qwant.com/*'],
```

Comment:
Could you limit the match patterns a bit more?
`['https://www.qwant.com/?*', 'https://lite.qwant.com/?*']` (which paths are `/?*`) seems sufficient to me.